src/borg/archiver.py

```diff
@@ -625,8 +625,10 @@ def _process(self, *, path, parent_fd=None, name=None,
             elif stat.S_ISDIR(st.st_mode):
                 with OsOpen(path=path, parent_fd=parent_fd, name=name, flags=flags_dir,
                             noatime=True, op='dir_open') as child_fd:
-                    with backup_io('fstat'):
-                        st = stat_update_check(st, os.fstat(child_fd))
+                    # child_fd is None for directories on windows. A race condition check is not possible
```

Comment:
- add a dot at the end of the sentence, please.
- also, to make it more clear "..., in that case a race condition ..."